src/eduid_common/api/decorators.py

```diff
@@ -89,6 +110,10 @@ def __call__(self, f):
         @wraps(f)
         def decorated_function(*args, **kwargs):
             ret = f(*args, **kwargs)
+
+            if isinstance(ret, Response):  # No need to Marshal again, someone else already did that
+                return ret
+
```

Comment:
Yeah, I spoke too soon. I was trying to be "all encompassing" but, as the module name implies, werkzeug.wrappers.Response does not inherit from webob.Resonse, it is a wrapper.
I have imported both, and done `insinstance(ret, (werkzeug_Response, webob_Response))` and now all test in eduid-webapp pass.